script/utils/InvestableWorkstreams/01-DeployInvestableWorkstream.s.sol

```diff
@@ -0,0 +1,273 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "forge-std/Script.sol";
+import "forge-std/Test.sol";
+
+import "../../deployment/DeploymentScript.s.sol";
+
+import {IFundingManager} from "src/modules/fundingManager/IFundingManager.sol";
+import {IModule} from "src/modules/base/IModule.sol";
+import {IOrchestratorFactory} from "src/factories/IOrchestratorFactory.sol";
+import {IOrchestrator} from "src/orchestrator/Orchestrator.sol";
+import {
+    BountyManager,
+    IBountyManager
+} from "src/modules/logicModule/BountyManager.sol";
+import {BancorVirtualSupplyBondingCurveFundingManager} from
+    "src/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.sol";
+
+import {BancorFormula} from
+    "src/modules/fundingManager/bondingCurveFundingManager/formula/BancorFormula.sol";
+
+import {ERC20} from "@oz/token/ERC20/ERC20.sol";
+//import {ERC20Mock} from "test/utils/mocks/ERC20Mock.sol";
+//import {ScriptConstants} from "../script-constants.sol";
+
+contract SetupInvestableWorkstream is Test, DeploymentScript {
+    //ScriptConstants scriptConstants = new ScriptConstants();
+    bool hasDependency;
+    string[] dependencies = new string[](0);
+
+    // ========================================================================
+    // ENVIRONMENT VARIABLES OR CONSTANTS
+
+    uint orchestratorOwnerPrivateKey =
+        vm.envUint("ORCHESTRATOR_OWNER_PRIVATE_KEY");
+    address orchestratorOwner = vm.addr(orchestratorOwnerPrivateKey);
+
+    address collateralTokenAddress =
+        vm.envAddress("BONDING_CURVE_COLLATERAL_TOKEN");
+    ERC20 collateralToken = ERC20(collateralTokenAddress);
+
+    address bancorFormulaAddress = vm.envAddress("BANCOR_FORMULA_ADDRESS");
+    BancorFormula formula = BancorFormula(bancorFormulaAddress);
```

Comment:
Curious if we store the address in the `.env` file manually or do we export them?